LaTeX/TeX.sublime-syntax

```diff
@@ -114,11 +169,70 @@ contexts:
         - include: macro-braces
 
   greeks:
-    - match: ((\\)(alpha|beta|gamma|delta|epsilon|varepsilon|zeta|eta|theta|vartheta|iota|gamma|kappa|lambda|mu|nu|xi|pi|varpi|rho|varrho|sigma|varsigma|tau|upsilon|phi|varphi|chi|psi|omega|Gamma|Delta|Theta|Lambda|Xi|Pi|Sigma|Upsilon|Phi|Psi|Omega))(?=\b|_)
+    - match: ((\\)({{greeks}}))(?=\b|_)
+      captures:
+        # keyword.other.greek.math.tex retained for backward compatibility
+        1: keyword.other.math.greek.tex keyword.other.greek.math.tex
+        2: punctuation.definition.backslash.tex
+
+  math-functions:
+    - match: ((\\)({{mathfun}}))(?=\b|_)
+      captures:
+        1: keyword.other.math.function.tex
+        2: punctuation.definition.backslash.tex
```

Comment:
Old TextMate syntaxes used many capture groups. Maybe some of them were added by converting to sublime-syntax format. We should use as little of them as possible to get best performance results.

```suggestion
  math-functions:
    - match: (\\){{mathfun}}(?=\b|_)
      scope: keyword.other.math.function.tex
      captures:
        1: punctuation.definition.backslash.tex
```